src/NHSDPublicBrowseAcceptanceTestsSpecflow/Features/FoundationSolutionCapabilityFilteredBrowse.feature

```diff
@@ -1,11 +0,0 @@
-﻿Feature: Foundation Solution (Single) Capability-filtered browse
```

Comment:
where is this covered if you're removing it?

Reply:
This was only ever a placeholder until more functionality was included. As it is now obsolete, I have removed it